satellite/client/client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package client
+
+import (
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
+
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/kademlia"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/pkg/transport"
+)
+
+var (
+	// ClientError wraps errors returned from client package
+	ClientError = errs.Class("satellite client error")
+)
+
+// SatelliteClient maintains variables required for talking to basic satellite endpoints
+type SatelliteClient struct {
+	log       *zap.Logger
+	transport transport.Client
+	kad       *kademlia.Kademlia
+}
+
+// New creates an Satellite Client
+func New(log *zap.Logger, identity *identity.FullIdentity, kad *kademlia.Kademlia) *SatelliteClient {
```

Comment:
Pass in `transport.Client` not `identity.FullIdentity`, most places should use that approach. Kademlia seems unnecessary, since we communicate with the satellite address directly.

Reply:
done